NBitcoin/NBitcoin.csproj

```diff
@@ -30,6 +30,9 @@
 	<PropertyGroup Condition=" '$(TargetFramework)' == 'net452' ">
 		<DefineConstants>$(DefineConstants);WIN;LEGACY_X509</DefineConstants>
 	</PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">

```

Comment:
why do we need a new target? .NETStandard should work fine no?

Reply:
I added that new target because for Linux and OSX the building script is:

`dotnet build -c Release -f netcoreapp2.0`

But TBH I don't remember why we need it.